sources/packages/backend/apps/workers/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -321,4 +332,108 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
       });
     }
   }
+
+  /**
+   * Get all outstanding student assessments to be calculated
+   * in the order of original assessment study start date.
+   * @param studentId student id.
+   * @param programYearId program year id.
+   * @returns student assessment to be calculated.
+   */
+  async getOutstandingAssessmentsForStudentInSequence(
```

Comment:
Do we need to return more than one record at this time or is it in preparation to some future implementation?